core/trino-main/src/test/java/io/trino/type/TestDecimalOperators.java

```diff
@@ -142,6 +142,10 @@ public void testSubtract()
     public void testMultiply()
```

Comment:
please add JMH benchmark results to the commit message. Please align before/after cases pairwise

Reply:
Added the benchmark results to the commit message; there's a lot of noise there, so it's not clear if they are very conclusive